projects/models.py

```diff
@@ -52,6 +52,18 @@ def is_overdue(self):
         today = date.today()
         return today > self.due_date and self.status != Project.ProjectStatus.COMPLETED
 
+    def set_to_done(self):
+        "Sets all the features associated with it to done, returns false if no features"
+        Project.objects.filter(id=self.id).update(status=30)
+        has_feature = False
+        data_f = Feature.objects.all()
```

Comment:
Similar comment, instead of querying the database for all Features, this is already inside the Project model which has a relationship to Features (OneToMany). You can directly access the Features on this Project through the `self` variable.

See here for the documentation:
https://docs.djangoproject.com/en/3.0/topics/db/examples/many_to_one/